test/e2e/features/story_openshift.feature

```diff
@@ -36,31 +36,48 @@ Feature: 3 Openshift stories
 		Then with up to "4" retries with wait period of "1m" http response from "http://httpd-example-testproj.apps-crc.testing" has status code "200"
 		And executing "oc delete project testproj" succeeds
 
-	# Old: Local image to image-registry feature
+	# Local image to image-registry feature
 
-	@linux @testdata
-	Scenario: Create local image, push to registry, deploy
-		Given checking that CRC is running
-		And executing "podman pull quay.io/centos7/httpd-24-centos7" succeeds
+	@darwin @linux @windows @story_registry
+	Scenario: Mirror image to OpenShift image registry
+		# mirror
 		When executing "oc new-project testproj-img" succeeds
-		And executing "podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.apps-crc.testing --tls-verify=false" succeeds
-		And executing "podman tag quay.io/centos7/httpd-24-centos7 default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test" succeeds
-		And executing "podman push default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test --tls-verify=false" succeeds
+		And  executing "oc registry login --insecure=true" succeeds
+		Then executing "oc image mirror registry.access.redhat.com/ubi8/httpd-24:latest=default-route-openshift-image-registry.apps-crc.testing/testproj-img/httpd-24:latest --insecure=true --filter-by-os=linux/amd64" succeeds
+		And executing "oc set image-lookup httpd-24" succeeds
+		# deploy
+		When executing "oc new-app testproj-img/httpd-24:latest" succeeds
+		When executing "oc rollout status deployment httpd-24" succeeds
+		Then stdout should contain "successfully rolled out"
+		When executing "oc get pods" succeeds
+		Then stdout should contain "Running"
+		When executing "oc logs deployment/httpd-24" succeeds
+		Then stdout should contain "httpd"
+		# cleanup
+		And executing "oc delete project testproj-img" succeeds
+
+	@darwin @linux @windows @local-img-to-registry @testdata @story_registry
+	Scenario: Pull image locally, push to registry, deploy
+		Given podman command is available
+		And executing "oc new-project testproj-img" succeeds
+		When pulling image "quay.io/centos7/httpd-24-centos7", logging in, and pushing image to registry succeeds
```

Comment:
May reword like:

_"When pushing local image "IMAGE_NAME" to internal registry"_

This will include all steps: login internal, pull image to local, tag and push to internal registry

Reply:
You could also create your local image instead of pulling it from somewhere, right? I don't mind changing it, just checking. For now, I've added `internal` to clarify what registry!